tests/phpunit/src/Commands/CodeStudio/CodeStudioWizardCommandTest.php

```diff
@@ -67,6 +67,7 @@ public function providerTestCommand(): array {
           '--key' => $this->key,
           '--secret' => $this->secret,
         ],
+        //'--phpVersion' => $this->phpVersion,
```

Comment:
@shubham-bansal96 Can these lines be removed instead of commented out. This option doesn't exist, so we do not need this.